includes/plugin-instrumentation.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+use Razorpay\Api\Api;
+use Razorpay\Api\Errors;
+
+class TrackPluginInstrumentation
+{
+    public $api;
+
+    public function __construct($key_id, $key_secret)
+    {
+        $this->api = new Api($key_id, $key_secret);
```

Comment:
@abdulwahidsharief Please check if you can use instance `getRazorpayApiInstance` instead re-defining api here.

Reply:
for some events have to get key and secret submitted by user so creating API object here